gen/decode.go

```diff
@@ -85,12 +85,14 @@ func (d *decodeGen) structAsTuple(s *Struct) {
 	d.p.declare(structArraySizeVar, u32)
 	d.assignAndCheck(structArraySizeVar, arrayHeader)
 	d.p.arrayCheck(strconv.Itoa(nfields), structArraySizeVar)
+	d.p.print("\n{")
```

Comment:
> I think the generated code would look a bit cleaner if we put
> `declare(...); assignAndCheck(...); arrayCheck(...)` inside a block,
> rather than putting each field in a block. 

True, updated.